tests/accessor/accessor_common.h

```diff
@@ -283,14 +353,15 @@ void read_write_zero_dim_acc(AccT testing_acc, ResultAccT res_acc) {
  * @tparam GetAccFunctorT Type of functor for accessor creation
  */
 template <accessor_type AccType, typename DataT, sycl::access_mode AccessMode,
-          sycl::target Target, typename GetAccFunctorT>
+          sycl::target Target = sycl::target::device, typename GetAccFunctorT>
```

Comment:
Nit: clang-format
```suggestion
          sycl::target Target = sycl::target::device,
          typename GetAccFunctorT>
```

Reply:
Are you sure, that we need wrapping to a new line here? The length of line 356 is 79 and it is not triggering `clang-format`

Reply:
Of course we do not, sorry)

I've had something like this locally due to a different base: https://github.com/KhronosGroup/SYCL-CTS/blob/1eeb805e6edaef53320121a9d1a54e1383757ace/tests/accessor/accessor_common.h#L462 
and somehow misinterpreted the direction of the local patch.